inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -55,6 +56,21 @@ std::string translate_type_name(const std::string& name) {
     return name;
 }
 
+// Some of the operators were added to wrong opsets. This is a
+// mapping that allows such operators to be serialized with proper
+// opsets. If new operators are discovered that have the same problem
+// the opset mapping needs to be updated here.
+const std::unordered_map<std::string, std::string>
+    special_operator_to_opset_assignments = {{"ShuffleChannels", "opset3"}};
```

Comment:
I think we need to check the type and the version of operation (type_info).

Reply:
We're checking type at line 239, if there isn't an op in this map with such name we're doing a normal search through opsets. As to op version, do you mean a possible situation where we add ShuffleChannelsV1 and we want to return different opset number?

Reply:
Yes I am talking about the situation if we have the `ShuffleChannels` with another version (for example from opset 200), without the version we will change opset version for all `ShuffleChannels` operations.

Reply:
Done.